Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:16
```

Comment:
I remember I had an issue with a specific node version, but not sure it was that version. @untzag, have you tested this image works locally?

@gwbischof, what version of node do you use?

Reply:
I have node v15.4.0. I don't remember what version that we had a problem with.